### PR TITLE
Update BuildingBricks and add HardModeTweaks, TwinTails, DulceDeLeche…

### PR DESCRIPTION
**Pull request by hea3ven**
… and Malleable

### CODE REVIEW

File: NEMP/mods.json [hunk 1]
```diff
@@ -4206,8 +4206,80 @@
...
+    },
+    "HardModeTweaks": {
+        "function": "CheckForgeJson",
+        "active": true,
+        "forgejson": {
+            "url": "https://raw.githubusercontent.com/hea3ven/HardModeTweaks/version/media/update.json",
+            "mcversion": "1.9"
+        }
+    },
+    "HardModeTweaks": {
```
Comment (NightKev) [inline NEMP/mods.json]: You can't have multiple mods with the same name (I'm surprised the JSON checker doesn't complain...). Name them after their MC version (ie: "HardModeTweaks-1.9.4").

Edit: Actually I'm not sure having multiple will work anyway due to the function being used... I think you'll just have to stick with the latest MC version or something.

Comment (hea3ven) [inline NEMP/mods.json]: I remember someone saying in irc that if you wanted to poll multiple versions you needed to add multiple entries, that's why I did it.

Comment (NightKev) [inline NEMP/mods.json]: Well assuming it works, you still need to give them different names.

Comment (SinZ163) [inline NEMP/mods.json]: You have seperate names, and use `"name" : "HardModeTweaks"` for it to map back to the correct mod name

Comment (SinZ163) [inline NEMP/mods.json]: https://github.com/NotEnoughMods/NotEnoughModPolling/blob/master/NotEnoughMods_Polling.py#L311-L314 for where the magic happens

Comment (hea3ven) [inline NEMP/mods.json]: Nice, I see, I've updated it with the "name" property and unique top level names.

**Pull request closed without merging**